src/utils/key.rs

```diff
@@ -261,10 +300,18 @@ mod test {
     fn test_rsa_key_operation() {
         let mut key_bundle = KeyBundle::new("rsa-2048", "rsa", 2048);
         test_key_sign_verify(&mut key_bundle);
+        test_key_encrypt_decrypt(&mut key_bundle, None);
```

Comment:
The `test_key_encrypt_decrypt` function's parameter `aad` also needs to rename to a proper one, to avoid misunderstanding and increase the readability of the code.

Reply:
> The `test_key_encrypt_decrypt` function's parameter `aad` also needs to rename to a proper one, to avoid misunderstanding and increase the readability of the code.

fixed